tests/func/test_commit.py

```diff
@@ -95,3 +97,13 @@ def test_commit_pipeline_stage(tmp_dir, dvc, run_copy):
     assert dvc.commit(f":{stage.addressing}") == [stage]
     assert dvc.commit(f"{PIPELINE_FILE}:{stage.addressing}") == [stage]
     assert dvc.commit(PIPELINE_FILE) == [stage]
+
+
+def test_imported_entries_unchanged(tmp_dir, dvc, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("file", "file content", "initial commit")
+
+    stage = dvc.imp(fspath(erepo_dir), "file")
+
+    with dvc.state:
+        assert stage.changed_entries() == ([], [], None)
```

Comment:
Won't it show up on `dvc.status`?

Reply:
IIRC these are not used by `status` but by `commit` to show prompt message on what's changed.

Reply:
@skshetry is right, in case of commit we check whether `changed_checksum` for `RepoDependency` changed. Upon status, we actually get the checksum for the file from the external repo, to get information on whether it has been updated.